Calling/ClientApp/src/components/LocalStreamMedia.tsx

```diff
@@ -23,7 +23,8 @@ export default (props: LocalStreamMediaProps): JSX.Element => {
     styles: {
       root: {
         width: '100%',
-        height: '100%'
+        height: '100%',
+        display: activeStreamBeingRendered ? 'none' : 'block'
```

Comment:
Looks more clean, just double check that this works as expected, since having it in the `style` could be intended to override other styles.

Reply:
Thanks for pointing this out! This prop actually overrides the "style" prop in the inline object. We're not building the hero sample for others to customize on top of it so I think this is fine.